zio-config/src/main/scala/zio/config/package.scala

```diff
@@ -20,10 +11,6 @@ package object config extends Sources {
       })
       .onError(_ => None)
 
-  def read[A](config: => Config[A]): Read[A]     = Read.read[A](config)
-  def write[A](config: => Config[A]): Write[A]   = Write.write[A](config)
-  def report[A](config: => Config[A]): Report[A] = Report.report[A](config)
```

Comment:
Based on the discussions, the read write and report are back in here. We can inline the functions later on though.
@wi101?

Reply:
Alright